### ISSUE TITLE
I cannot launch Minecraft

### ISSUE THREAD

**Issue by Calvados1412** [killpowa/Twickt-Launcher#1244]
Good Day,
since yesterday i can't start Minecraft through the GDLauncher. 

**To Reproduce**
I do not think it's possible to reproduce, as it works with my friends. Whenever I try to start the Modpack (All The Mods 7, 0.2.19, version 1.18.1), it gets stuck in the ,,Your instance is starting'' screen. It starts normally with every other mod.

**Expected behavior**
I would like to start it again, as we have put much effort in this world

**Possible solutions**
Maybe it works with a updated version of the mod, can't try it yet. 

**Screenshots**

**Operating System:**
- OS: Windows
- Version: 10
- Java Version: 
- Minecraft Version: 1.18.1
- Mod Loader: Forge

**Comment by Eskaan** [comment]
Could you show us your console (>_ at the top) and Anny errors and other things showing there, please?
Maybe it’s a simple fault on side of the Modpack / a corrupted instance.

**Comment by Calvados1412** [comment]
![Developer Tools - file____C__Users_Paul_AppData_Local_Programs_gdlauncher_resources_app asar_build_index html 05 01 2022 20_41_39_LI](https://user-images.githubusercontent.com/97187132/148278873-eed8e430-9add-48f9-8d61-32d359348273.jpg)

**Comment by Eskaan** [comment]
Yes, and now open the console tab. And then, after trying to start the instance, send us a screenshot
